libtransact/src/workload/batch_gen.rs

```diff
@@ -162,17 +162,19 @@ mod tests {
 
     use cylinder::{secp256k1::Secp256k1Context, Context, Signature, Signer};
 
-    use crate::protos::batch::BatchHeader;
-    use crate::protos::transaction::TransactionHeader;
+    use crate::protocol::batch::BatchHeader;
+    use crate::protocol::transaction::{HashMethod, TransactionBuilder};
+    use crate::protos::transaction::Transaction as ProtoTransaction;
```

Comment:
Call this ProtobufTransaction if you take my other suggestions for ProtocolBatch naming, etc.

Reply:
Fixed